config/locals.go

```diff
@@ -132,6 +132,8 @@ func attemptEvaluateLocals(
 	errs := &errors.MultiError{}
 
 	for _, attr := range attrs {
+		l.Debugf("TRACE: evaluating local %q...", attr.Name)
```

Comment:
Instead of the "trace" prefix, it might be better to use a [trace-level message](https://github.com/gruntwork-io/terragrunt/blob/64bd38a08274f7f5593325041bd6589f028f58cd/pkg/log/logger.go#L63)?